SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -1535,6 +1539,10 @@ void CFVMFlowSolverBase<V, R>::EdgeFluxResidual(const CGeometry *geometry,
     InstantiateEdgeNumerics(solvers, config);
   }
 
+#ifdef HAVE_OPDI
+  const auto preaccEnabled = ReducerStrategy && AD::PausePreaccumulation();
+#endif
```

Comment:
We will do it like this to avoid different logic between primal and adjoint.
I have updated the warning message to reflect the higher memory usage in case the "ReducerStrategy" is used.
The places I modified now are also the places where the adjoint evaluation mode that does not use atomics could in theory be enabled.

Reply:
Don't you find short-circuiting hard to read?